sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -2325,6 +2330,7 @@ object Radar {
         limit: Int? = null,
         country: String? = null,
         expandUnits: Boolean? = null,
+        mailable: Boolean? = null,
```

Comment:
Similar to iOS, let's have a deprecated function heading and a revised one. The new one should have both `expandUnits` and `mailable` (this one here), but with `expandUnits: Boolean? = true` (b/c it's implicitly default true) and a note in the comment that it's always true / units are always returned.

The deprecated one just calls `RadarApiClient.autocomplete` without the `mailable` param.